cli/running/running.go

```diff
@@ -17,9 +18,11 @@ const (
 		`Make sure you ran "cartridge build" before running "cartridge start"`
 )
 
-func FillCtx(ctx *context.Ctx, args []string) error {
+func FillCtx(ctx *context.Ctx, cmd *cobra.Command, args []string) error {
 	var err error
 
+	ctx.Running.StateboardFlagIsSet = cmd.Flags().Changed("stateboard")
```

Comment:
This code is cobra-specific, I suggest moving this to `cli/commands`. You can create some function in `cli/commands/common.go` and call it in `cli/commands/{start,stop,status,log}.go`.